include/topic_based_ros2_control/topic_based_system.hpp

```diff
@@ -97,8 +97,7 @@ class TopicBasedSystem : public hardware_interface::SystemInterface
   double trigger_joint_command_threshold_ = 1e-5;
   // If the difference between the current joint state and joint command is more than this value,
   // the joint command will not be published.
-  double block_joint_command_threshold_ = 5;
-
+  double block_joint_command_threshold_;
```

Comment:
Maybe provides a default value here, or it will be undefined behavior if this value never get initialized.